onmt/transforms/misc.py

```diff
@@ -134,6 +136,100 @@ def apply(self, example, is_train=False, stats=None, **kwargs):
             raise ValueError(f'prefix for {corpus_name} does not exist.')
         return self._prepend(example, corpus_prefix)
 
+    def apply_reverse(self, translated):
+        def _removeprefix(s, prefix):
+            if s.startswith(prefix):
+                return s[len(prefix):]
```

Comment:
Might not be critical but you will retain a whitespace here, no? (len(prefix)+1?)

Reply:
good point. will fix this.